dimagi/pages/static/style/pages/sectors/reproductive_health.scss

```diff
@@ -0,0 +1,117 @@
+@import 'app/mixins/background';
+@import 'app/variables/colors';
+@import 'app/variables/sizes';
+@import 'app/variables/type';
+
+#page-intro {
+  $_base-height: 190px;
+  $_height-by-size: (
+    small: 280px,
+    medium: 280px,
+    large: 280px,
+    extra-large: 320px,
+  );
+  @include make-bg(hero-data-image, '../../../pages/sectors/reproductive-health/intro/hero', $_base-height, $_height-by-size);
+}
+
+.news-images .image.blog1 {
+  @include background-cover('../../../pages/sectors/reproductive-health/news/Blog1.jpg');
+  height: 200px;
+  background-size: contain;
+  background-position: left;
+}
+
+.news-images .image.blog2 {
+  @include background-cover('../../../pages/sectors/reproductive-health/news/Blog2.jpg');
+  height: 200px;
+  background-size: contain;
+  background-position: left;
+}
+
+.news-images .image.blog3 {
+  @include background-cover('../../../pages/sectors/reproductive-health/news/Blog3.jpg');
+  height: 200px;
+  background-size: contain;
+  background-position: left;
+}
+
+.programs-images .image.featured-project1 {
+  @include background-cover('../../../pages/sectors/reproductive-health/programs/FeaturedProject1.jpg');
+  height: 200px;
+  background-size: contain;
+  background-position: left;
+}
+
+.programs-images .image.featured-project2 {
+  @include background-cover('../../../pages/sectors/reproductive-health/programs/FeaturedProject2.jpg');
+  height: 200px;
+  background-size: contain;
+  background-position: left;
+}
+
+.programs-images .image.featured-project3 {
+  @include background-cover('../../../pages/sectors/reproductive-health/programs/FeaturedProject3.jpg');
+  height: 200px;
+  background-size: contain;
+  background-position: left;
+}
+
+.modal-popup {
+  width: 309px !important;
+}
+
+.part3 {
+  $_base-height: 550px;
+  $_height-by-size: (
+    small: 620px,
+    medium: 385px,
+    large: 360px,
+    extra-large: 360px,
+  );
+  @include make-bg(epic-cover, '../../../pages/sectors/reproductive-health/part3/hero', $_base-height, $_height-by-size);
+}
+
+.center {
+  display: block;
+  margin-left: auto;
+  margin-right: auto;
+}
+
+.commcare-text {
+  $_base-height: 290px;
+  $_height-by-size: (
+    small: 290px,
+    medium: 290px,
+    large: 290px,
+    extra-large: 290px,
+  );
+  @include make-bg(phone-image, '../../../pages/sectors/reproductive-health/part3/Phone', $_base-height, $_height-by-size, '.png');
+
+  @media (min-width: $width-desktop) {
+    .off-grid {
+      top: 10%;
+      left: 0;
+      position: absolute;
+      right: auto;
+      width: 120%;
+      z-index: 1;
+    }
+
+    .content {
+      position: relative;
+      z-index: 2;
+    }
+  }
+}
+
+.part3 .image .bg-loader{
+  background-color: transparent;
+}
+
+.mb-10 {
+  margin-bottom: 10px;
+}
+
+.read-more:hover,.read-more:focus{
+  color: #004ebc!important;
```

Comment:
see comment I made about colors in https://github.com/dimagi/dimagi.com/pull/92
also, styles separated by commas should be on new lines...eg:
```
.read-more:hover,
.read-more:focus {

}
```
There should always be a space between the last character of the class and the `{`. [Please enable scss-lint](https://github.com/sds/scss-lint) in your editor to avoid these really, really simple mistakes. There is a scss lint config file in the root directory of this project. Please use it.

Also, please comment why `!important` needs to be used here.